src/v2i-hub/TimPlugin/src/TimPlugin.h

```diff
@@ -104,7 +105,17 @@ class TimPlugin: public PluginClient {
 	void OnStateChange(IvpPluginState state);
 
 	bool TimDuration();
```

Comment:
Is this method used anywhere in the new plugin. If not we should remove it and remove the start and end time configuration parameters

Reply:
removed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/508/commits/c781252ca54d7d7f6442ee975c055d79f6bd19dc